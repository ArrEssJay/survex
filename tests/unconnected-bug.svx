; pos=fail
*fix 1 0 0 0
.. 2 1.00 000 0
2 .. 1.00 000 0
