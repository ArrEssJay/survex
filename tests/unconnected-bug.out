
Removing trailing traverses...

Concatenating traverses...

Simplifying network...
cavern: error: Survey not all connected to fixed stations
